R/import_gtfs.R

```diff
@@ -391,11 +386,10 @@ append_file_ext = function(file) {
       # behaviour defined in test_import_gtfs.R#292
       file_ext <- "txt"
     }
-    if(endsWith(.f, paste0(".", file_ext))) {
-      return(.f) # file extension already present
-    } else {
-      return(paste0(.f, ".", file_ext))
+    if (!identical(fs::path_ext(.f), file_ext)) {
```

Comment:
Why not `assert_extension`? Just curious.

Reply:
Thanks, i missed that one. Now updated.